src/sybil/table.go

```diff
@@ -26,13 +26,13 @@ type Table struct {
 	// List of new records that haven't been saved to file yet
 	newRecords RecordList
 
-	keyStringIdLookup map[int16]string
-	valStringIdLookup map[int32]string
+	keyStringIDLookup map[int16]string
+	valStringIDLookup map[int32]string
 
 	// This is used for join tables
 	joinLookup map[string]*Record
 
-	stringIdM *sync.RWMutex
+	stringIDM *sync.RWMutex
```

Comment:
stringIDM is weird, imo. dunno what's better, though - maybe something like... stringID_m?

Reply:
yeah agree. stringIdMu ok? I can rename other mutexes to follow that pattern.

Reply:
stringIDMu *